packages/smooth_app/lib/cards/product_cards/smooth_product_base_card.dart

```diff
@@ -179,7 +180,7 @@ class _SmoothProductCardHeader extends StatelessWidget {
               ),
               SizedBox(width: dense ? BALANCED_SPACE : MEDIUM_SPACE),
               Expanded(
-                child: Text(
+                child: AutoSizeText(
```

Comment:
Not sure AutoSizeText resizes a text when the overflow parameter set to ellipsis.